Find a functionally equivalent code:com.android.systemui.usb.StorageNotification.setUsbStorageNotification:COMMENT
Method Modifier: private     
Comment:/**
 * Sets the USB storage notification.
 */

@@ -26,32 +26,35 @@
             Intent intent = new Intent();
             pi = PendingIntent.getBroadcastAsUser(mContext, 0, intent, 0, UserHandle.CURRENT);
         }
+        mUsbStorageNotification.color = mContext.getResources().getColor(com.android.internal.R.color.system_notification_accent_color);
         mUsbStorageNotification.setLatestEventInfo(mContext, title, message, pi);
+        mUsbStorageNotification.visibility = Notification.VISIBILITY_PUBLIC;
+        mUsbStorageNotification.category = Notification.CATEGORY_SYSTEM;
         final boolean adbOn = 1 == Settings.Global.getInt(mContext.getContentResolver(), Settings.Global.ADB_ENABLED, 0);
         if (POP_UMS_ACTIVITY_ON_CONNECT && !adbOn) {
             // Pop up a full-screen alert to coach the user through enabling UMS. The average

